v-next/example-project/scripts/deploy-rocket-from-script.ts

```diff
@@ -14,7 +14,7 @@ const name = await apollo.read.name();
 const status = await apollo.read.status();
 
 console.log(
-  `Deployed rocket with Ignition from a Hardhat Script 🚀
+  `Deployed rocket with Ignition and Viem from a Hardhat Script 🚀
```

Comment:
isn't this ethers?

Reply:
Sorry, this is a change in the `viem` script in the example project. I initially added an equivalent ethers script and this was a change to help with differentiation. I think we should keep it.